docker/ml-with-gpu/Dockerfile

```diff
@@ -17,31 +17,31 @@ WORKDIR /data
 
 # Note that CUDA Toolkit version might work with only a particular version of
 # NVidia drivers, e.g. cuda 9.2 needs drivers 396.37 and cuda 9.0 needs 384.130.
-# Drivers are installed in
-# clusterfuzz/configs/gce/linux-init-internal-ml-with-gpu.yaml.
+# The drivers are installed in
+# configs/test/gce/linux-init-ml-with-gpu.yaml.
 # cuDNN packages installed later in this script must match the CUDA version too.
 
-# From https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&target_distro=Ubuntu&target_version=1604&target_type=deblocal
-RUN wget https://developer.nvidia.com/compute/cuda/9.0/Prod/local_installers/cuda-repo-ubuntu1604-9-0-local_9.0.176-1_amd64-deb -O cuda.deb && \
-    dpkg -i cuda.deb && \
-    apt-key add /var/cuda-repo-9-0-local/7fa2af80.pub && \
-    apt-get update && \
-    apt-get install -y cuda
-
-# From (requires registration) https://docs.nvidia.com/deeplearning/sdk/cudnn-install/#installlinux-deb
-RUN wget https://storage.googleapis.com/clusterfuzz-data/cudnn/7.1.4_cuda9.0/libcudnn7_7.1.4.18-1%2Bcuda9.0_amd64.deb && \
-    wget https://storage.googleapis.com/clusterfuzz-data/cudnn/7.1.4_cuda9.0/libcudnn7-dev_7.1.4.18-1%2Bcuda9.0_amd64.deb && \
-    wget https://storage.googleapis.com/clusterfuzz-data/cudnn/7.1.4_cuda9.0/libcudnn7-doc_7.1.4.18-1%2Bcuda9.0_amd64.deb && \
-    dpkg -i libcudnn7_*.deb && \
-    dpkg -i libcudnn7-dev_*.deb && \
-    dpkg -i libcudnn7-doc_*.deb
+# The dependencies installation commands below follow
+# https://www.tensorflow.org/install/gpu#ubuntu_1604_cuda_101.
+
+# Install development and runtime libraries (~4GB)
+RUN apt-get install -y --no-install-recommends \
+    cuda-10-1 \
+    libcudnn7=7.6.4.38-1+cuda10.1  \
+    libcudnn7-dev=7.6.4.38-1+cuda10.1
+
+# Install TensorRT. Requires that libcudnn7 is installed above.
+RUN apt-get install -y --no-install-recommends \
+    libnvinfer6=6.0.1-1+cuda10.1 \
```

Comment:
nit: CUDA_VERSION=6.0.1-1+cuda10.1 at start of command and then use in three places as $CUDA_VERSION ?
Same for libcudnn7 above.

Reply:
thanks for the suggestion! I don't think that would improve readability a lot, plus it's not CUDA version (cuda is 10.1), so I'm just leaving this like suggested in the docs: https://www.tensorflow.org/install/gpu#ubuntu_1604_cuda_101